app/src/main/java/upday/droidconmvvm/mvvm/MVVMActivity.java

```diff
@@ -0,0 +1,75 @@
+package upday.droidconmvvm.mvvm;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView;
+
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+import rx.subscriptions.CompositeSubscription;
+import upday.droidconmvvm.DroidconApplication;
+import upday.droidconmvvm.R;
+import upday.droidconmvvm.datamodel.IDataModel;
+
+public class MVVMActivity extends AppCompatActivity {
+
+    @NonNull
+    private CompositeSubscription mSubscription;
+
+    @NonNull
+    private ViewModel mViewModel;
+
+    @Nullable
+    private TextView mGreetingView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        mViewModel = new ViewModel(getDataModel());
+        setupViews();
+    }
+
+    private void setupViews() {
+        mGreetingView = (TextView) findViewById(R.id.greeting);
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        bind();
+    }
+
+    @Override
+    protected void onPause() {
+        super.onPause();
+        unBind();
+    }
+
+    private void bind() {
+        mSubscription = new CompositeSubscription();
+
+        mSubscription.add(mViewModel.getGreeting()
+                                    .subscribeOn(Schedulers.computation())
```

Comment:
In presenter you do not use subscribeOn/observeOn, so you could either remove it from here or add it there